cpp/eudaq/src/JadeRawEvent2StdEventConverter.cc

```diff
@@ -39,7 +39,7 @@ bool JadeRawEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::StdEv
   const std::vector<uint16_t> &data_x = df.Data_X();
   const std::vector<uint16_t> &data_y = df.Data_Y();
   const std::vector<uint16_t> &data_z = df.Data_D();
-
+  int plane_id = ev->GetStreamN();
```

Comment:
FYI:
GetStreamN is the EUDAQ_ID in the Producer section of conf file. If not set, it is random. 
GetExtension is set by the alpide producer with plane number (which is binded to daqboard ip address).

Reply:
Good to know - we can maybe discuss offline how we should handle this best. Currently, I think the config files do match.